### PR TITLE
rm 2020 application link

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

**Comment by jnewbery** [comment]
Pushed some suggested changes.

I think "(COVID-19 dependent)" without instructions on where to find more information raises more questions than it answers, so I removed it from the head and added more text to a banner below.

**Comment by adamjonas** [comment]
I'd prefer to pull the dates from the banner if we aren't going to have the COVID announcement above the fold. If you want to do that I'm fine to merge after that. I think it should go out today.

**Comment by jnewbery** [comment]
done. If you're happy, I can squash/merge.

File: index.html [hunk 1]
```diff
@@ -977,7 +969,8 @@ <h4>Samuel Dobson</h4>
       <div class="container">
         <div class="row justify-content-center">
           <div class="col-md-4">
-		  <span class="copyright">Copyright &copy; <a href="https://chaincode.com">Chaincode Labs</a> 2020</span>
+            <span class="copyright">Copyright &copy; <a href="https://chaincode.com">Chaincode Labs</a> 2020</span>
+            <a href="residency@chaincode.com">residency@chaincode.com</a>
```
Comment (adamjonas) [inline index.html]: This shouldn't be a link. It's an email. Right now that would click through to residency.chaincode.com/residency@chaincode.com.

I'd prefer a span as found on the resources page.

File: resources.html [hunk 2]
```diff
@@ -82,10 +82,6 @@
           <div class="intro-heading">Chaincode Residency 2020</div>
           <div class="intro-lead-in">Lightning: June 1st - 5th</div>
```
Comment (adamjonas) [inline resources.html]: Probably worth pulling the dates from here as well.

File: index.html (hunk 1 above)
Comment (jnewbery) [reply index.html]: done

File: resources.html (hunk 2 above)
Comment (jnewbery) [reply resources.html]: done

**Comment by jnewbery** [comment]
applied the suggested changes, squashed and pushed

**Pull request merged**